README.md

```diff
@@ -141,36 +154,41 @@ yarn start # start the dev server
 
 A development server should now be running on [localhost:4000](localhost:4000).
 
-## Step 2. View your Blocks within a sandbox
+## Step 2. View your development server within the Blocks app
 
-When you visit [localhost:4000](localhost:4000) in your browser, you'll see an interface where you can test out your Block.
+When you visit [localhost:4000](localhost:4000) in your browser, you'll be
+redirected to the Blocks app with your locally-developed blocks embedded in it.
+You can see your blocks by clicking on the block picker toward the top of the
+page (they're shown at the top in blue with a plug icon):
 
-This starter project has one example folder block and one example file block.
+<img src="https://user-images.githubusercontent.com/56439/181648955-101b6567-3f9b-44b3-af99-7ef3ca6161b9.png" width="418" />
 
-[Sandbox video](https://user-images.githubusercontent.com/8978670/144666304-b9012177-aed6-4c26-afc3-3de8c8d6d0ad.mov)
-
-You can play with two bits of the interface to view your Blocks:
-
-1. **Input file or folder**: An input that accepts the path to a file or a folder on github.com. Here are some valid file/folder paths.
+This starter project has one example folder block and one example file block.
+Try choosing one of the example blocks in the block picker to see it in action.
+When you make changes to the block code it will be hot-reloaded in the app.
 
-- https://github.com/facebook/react/blob/main/README.md (file)
-- https://github.com/facebook/react/blob/0.3-stable/README.md (file)
-- https://github.com/facebook/react/blob/main/packages/react-dom/index.js (file)
-- https://github.com/facebook/react/ (folder)
-- https://github.com/facebook/react/tree/0.3-stable (folder)
-- https://github.com/facebook/react/tree/main/packages (folder)
+If you don't see your blocks, make sure that:
 
-2. **List of custom blocks**: A menu of the blocks that you have defined in `blocks.config.json` (see below; for now this contains the example file and folder blocks)
+- your repo is public
+- your development server is running
+- there's a `devServer` query parameter in the URL pointing to your development server
```

Comment:
might be worth throwing in the default link: https://blocks.githubnext.com/githubnext/blocks-tutorial?devServer=http://localhost:4000

Reply:
👍